code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -5641,4 +5641,16 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
 	name = "Hyoscyamine"
 	id = HYOSCYAMINE
 	description = "Hyoscyamine is a tropane alkaloid which can disrupt the central nervous system."
+	color = "#C8A5DC" //rgb: 200, 165, 220
```

Comment:
I actually haven't changed the colors of any of these from their parents.
I still intend to, though.